.github/workflows/repo-check-docker-build-ecr-deploy.yml

```diff
@@ -83,20 +83,23 @@ jobs:
       - name: Split registry
         id: registry
         run: |
-         echo "image-name=$(cut -d'/' -f2 <<< ${{inputs.ecr_registry_path}})" >> $GITHUB_OUTPUT
-         echo "ecr-registry-path=$(cut -d'/' -f1 <<< ${{inputs.ecr_registry_path}})" >> $GITHUB_OUTPUT
+          echo "image-name=$(cut -d'/' -f2 <<< ${{inputs.ecr_registry_path}})" >> $GITHUB_OUTPUT
+          echo "ecr-registry-path=$(cut -d'/' -f1 <<< ${{inputs.ecr_registry_path}})" >> $GITHUB_OUTPUT
       - name: login
         uses: aws-actions/configure-aws-credentials@v3
         with:
-           role-to-assume: ${{ inputs.aws-role }}
-           aws-region: eu-west-2
-           role-session-name: role-session-name-${{ github.actor }}-${{ github.triggering_actor }}
+          role-to-assume: ${{ inputs.aws-role }}
+          aws-region: eu-west-2
+          role-session-name: role-session-name-${{ github.actor }}-${{ github.triggering_actor }}
       - name: build and push
-        uses: nationalarchives/da-tre-github-actions/.github/actions/docker-build-and-deploy-to-ecr@0.0.10
+        uses: nationalarchives/da-tre-github-actions/.github/actions/docker-build-and-deploy-to-ecr@0.0.45
```

Comment:
I might be wrong but isn't this supposed to call `docker-build-and-ecr-deploy-with-wiz-scan.yml`

Reply:
I also updated docker-build-and-deploy-to-ecr  because it was used by the python and c# workflows and simplifying these would have been harder